examples/gin/main.go

```diff
@@ -0,0 +1,129 @@
+package main
+
+import (
+	"encoding/json"
+	"io/ioutil"
+	"net/http"
+	"os"
+	"path/filepath"
+	"strings"
+	"time"
+
+	"github.com/airbrake/gobrake/v5"
+	ginbrake "github.com/airbrake/gobrake/v5/gin"
+	"github.com/gin-gonic/gin"
+)
+
+var ProjectId int64 = 105138                               // Insert your Project Id here
+var ProjectKey string = "fd04e13d806a90f96614ad8e529b2822" // Insert your Project Key here
```

Comment:
Change many digits of the project key so that it's definitely not valid. Also, change the project id to something like `999999` so that it's more obvious that the user should change it.